.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+known_third_party = PIL,atomicwrites,holoviews,ipykernel,matplotlib,nbconvert,numpy,pytest,scipy,setuptools,skopt,sortedcollections,sortedcontainers,zmq
```

Comment:
what does this do?

Will this mean that if we ever add another library to the dependencies we'll need to add it here too?

Reply:
`isort` isn't very good at figuring out what libraries are third party libs. The pre-commit hook `seed-isort-config` automatically maintains this file.

Reply:
`isort` is [able to deduce the requirements from `requirements.txt`](https://github.com/timothycrosley/isort/blob/c23f76f8ed020455a78d9480ef9e52f476cf601b/isort/finders.py#L306-L317). We could also define that and get rid of the `.isort.cfg` file.

Reply:
I was under the impression that `requirements.txt` was really for deploying apps, and not really for library development.

How does `seed-isort-config` work?

Reply:
It [reads the source files](https://github.com/asottile/seed-isort-config/blob/master/seed_isort_config.py) IIUC.

Reply:
If `seed-isort-config` maintains this file, then we shouldn't check it in to git, right? It will just be autogenerated by `pre-commit`

Reply:
It's not possible to not commit the file:
```bash
(base) basnijholt-macbook  ➜  adaptive git:(power-up-pre-commit) ✗  rm .isort.cfg
(base) basnijholt-macbook  ➜  adaptive git:(power-up-pre-commit) ✗  pre-commit run --all
Trim Trailing Whitespace.................................................Passed
Fix End of Files.........................................................Passed
Check docstring is first.................................................Passed
Check Yaml...............................................................Passed
Debug Statements (Python)................................................Passed
Check python ast.........................................................Passed
flake8...................................................................Passed
black....................................................................Passed
pyupgrade................................................................Passed
isort....................................................................Passed
seed isort known_third_party.............................................Failed
hookid: seed-isort-config

Files were modified by this hook. Additional output:

Creating an .isort.cfg with a known_third_party setting. Feel free to move the setting to a different config file in one of .editorconfig, .isort.cfg, setup.cfg, tox.ini, pyproject.toml.

This setting should be committed.
```

Reply:
never mind; I just tried it locally and it complains that `.isort.cfg` should be committed.